uaclient/jobs/eol_status.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Dict
+
+from uaclient import messages, util
+from uaclient.config import UAConfig
+
+
+def get_os_release_vars() -> Dict:
+    vars = dict()
+    content = util.load_file("/etc/os-release")
+    for line in content.splitlines():
+        k, v = line.rstrip().split("=")
+        vars[k] = v.strip('"')
+    return vars
+
+
+def check_eol_and_update(cfg: UAConfig) -> bool:
+    if cfg.is_attached:
+        False
```

Comment:
```suggestion
        return True
```

Reply:
Thanks for this. I was not sure what to return at this point.